src/current/releases/index.md

```diff
@@ -155,29 +155,29 @@ The following releases and their descriptions represent proposed plans that are
       Determine if the major version is LTS and the patch component of the initial LTS patch,
       or the major version is a skippable innovation release
     {% endcomment %}
+    {% assign released = false %}
     {% assign has_lts_releases = false %}
     {% assign lts_link_linux = '' %}
     {% assign lts_patch = nil %}
     {% assign in_lts = false %}
     {% assign comparison = nil %}
     {% assign skippable = false %}
-    {% if v.initial_lts_patch != "N/A" %}
-        {% assign has_lts_releases = true %}
-        {% assign lts_link = '&nbsp;(<a href="release-support-policy.html">LTS</a>)&nbsp;' %}
-        {% capture lts_patch_string %}{{ v.initial_lts_patch | split: '.' | shift | shift }}{% endcapture %}
-        {% assign lts_patch = lts_patch_string | times: 1 %}{% comment %}Cast string to integer {% endcomment %}
-    {% elsif v.release_date != "N/A" and v.maint_supp_exp_date != "N/A" and v.asst_supp_exp_date == "N/A" %}
-        {% assign skippable = true %}
+    {% if v.release_date != "N/A" and v.maint_supp_exp_date != "N/A" %}
```

Comment:
It's an extra check to make sure something is not off with the metadata that determines support. I can't. check `asst_supp_exp_date` like we used to, because innovation releases remain at `N/A` for that field.

Reply:
SGTM. We're on the same page that it's an extra check. Can't hurt / can only help.